audition/regrets.py

```diff
@@ -0,0 +1,92 @@
+import copy
+
+
+def highest_metric_value(df, metric, param):
+    """Pick the model group with the highest metric value
+
+    Arguments:
+        metric (string) -- model evaluation metric, such as 'precision@'
+        metric_param (string) -- model evaluation metric parameter,
+            such as '300_abs'
+        df (pandas.DataFrame) -- dataframe that is keyed on model group id,
+            containing the columns:
+                model_id,
+                train_end_time,
+                metric,
+                parameter,
+                raw_value,
+                below_best,
+                below_best_next_time
+    Returns: (int) the model group id to select, with highest raw metric value
+    """
+    return df[df['raw_value'] == df['raw_value'].max()].index.tolist()[0]
```

Comment:
True. Looks like you've taken care of this (and much much more!) in your PR, so I'll leave this be for now.